datajoint/erd.py

```diff
@@ -208,7 +208,11 @@ def _make_graph(self):
             graph = nx.DiGraph(self).subgraph(nodes)
             nx.set_node_attributes(graph, 'node_type', {n: _get_tier(n) for n in graph})
             # relabel nodes to class names
-            mapping = {node: (lookup_class_name(node, self.context) or node) for node in graph.nodes()}
+            mapping = {node: (node) for node in graph.nodes()}
+            clean_context = dict((k, self.context[k]) for k in self.context
+                                 if '_' not in k)  # hack for ipython '_' var
```

Comment:
will acknowledge this could be a bit broad of a match even though we prefer *CamelCase* class names..

alternately could e.g.: ```if not re.match('^_*', k)``` or similar...